src/std/amb-test.ss

```diff
@@ -21,8 +21,8 @@
         (let ((baker (amb 1 2 3 4 5))
               (cooper (amb 1 2 3 4 5))
               (fletcher (amb 1 2 3 4 5))
-              (miller (amb 1 2 3 4 5))
-              (smith (amb 1 2 3 4 5)))
+              (miller (element-of '(1 2 3 4 5)))
+              (smith (element-of '(1 2 3 4 5))))
```

Comment:
I think we should leave this test as is (but it needs wrapping in a `begin-amb`) and write a couple more test cases for the various things we might want to test (like `element-of` but also `all-of` and `one-of`).